src/Webpack/ConfigFactory.ts

```diff
@@ -295,11 +291,7 @@ export default class WebpackConfigFactory {
                     // new VirtualModulesPlugin(virtualModules)
                 ],
                 resolve: {
-                    ...(commonConfig.resolve || {}),
-                    plugins: [
-                        ...(commonConfig.resolve && commonConfig.resolve.plugins || []),
-                        new MagentoThemeFallbackResolverPlugin(project, theme)
-                    ]
+                    ...(commonConfig.resolve || {})
```

Comment:
This would actually break our react projects, got sick of adding it to the green-pistachio config as a tap.